internal/worker/rundynamic.go

```diff
@@ -108,6 +110,28 @@ func addSSHKeysToSandbox(ctx context.Context, sb sandbox.Sandbox) error {
 	return sb.CopyIntoSandbox(ctx, tempdir+"/.", "/root/.ssh")
 }
 
+// generateAWSAccessID generates an AWS access key id based off of some known patterns and random values.
+func generateAWSAccessKeyId() (string, error) {
+	const charSet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ234567"
+	var accessKeyId = "AKIAI"
+	for i := 0; i < 14; i++ {
+		randInt, err := rand.Int(rand.Reader, big.NewInt(int64(len(charSet))))
+		if err != nil {
+			return "", err
+		}
+		accessKeyId += string(charSet[randInt.Int64()])
+	}
+	accessKeyId += "Q"
+	return accessKeyId, nil
+}
+
+// generateAWSSecretAccessKey generates a random 30 byte base64 encoded string to use as an AWS secret access key.
+func generateAWSSecretAccessKey() string {
+	b := make([]byte, 30)
+	rand.Read(b)
```

Comment:
see above, let's replace this call with [`rand.Read()`](https://pkg.go.dev/math/rand#Rand.Read) from `math/rand`, and merge this function with the one above